pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -106,6 +107,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 				obj.Logging.Format = "text"
 			}
 			obj.EnableSystemLogHandler = true
+			obj.MemoryThrottlingFactor = utilpointer.Float64Ptr(0.8)
```

Comment:
why a constant and not random?

Reply:
https://github.com/kubernetes/enhancements/blame/master/keps/sig-node/2570-memory-qos/README.md#L95

> e.g. if a container has `requests.memory=50, limits.memory=100`, and we have a throttling factor of .8, `memory.high` would be 80. if a container had no memory limit specified, we substitute `limits.memory` for `node allocatable memory` and apply the throttling factor of .8 to that value.


According to the kep definition, the default value is 0.8. This is based on practical experience.

Reply:
The point of a fuzzer is to try unexpected values.

Reply:
Thanks for reminding, have updated.